internal/handlers/scan_sbom.go

```diff
@@ -129,6 +137,13 @@ func (h *ScanSBOMHandler) Handle(message messaging.Message) error {
 	}
 
 	_, err = controllerutil.CreateOrUpdate(ctx, h.k8sClient, vulnerabilityReport, func() error {
+		if vulnerabilityReport.Labels == nil {
+			vulnerabilityReport.Labels = prepareVulnerabilityReportLabels()
+		} else {
+			vulnerabilityReport.Labels["app.kubernetes.io/managed-by"] = LabelValueManagedBy
+			vulnerabilityReport.Labels["app.kubernetes.io/component"] = LabelValueComponent
+			vulnerabilityReport.Labels["app.kubernetes.io/part-of"] = LabelValuePartOf
+		}
```

Comment:
I've seen the comment of @holyspectral about the risk of overwriting use labels. In this case however, we are the sole owners of the `VulnerabilityReport` resource, the user is not expected to add custom labels to it.

I would just set the labels we expect. We can always change that in the future if there's actual demand from users.

BTW, we were to keep the user labels around, a better way to do that would have been:

```golang
if vulnerabilityReport.Labels == nil {
  vulnerabilityReport.Labels = make(map[string]string)
}

vulnerabilityReport.Labels[constants.LabelManagedByKey] = constants.LabelManagedByValue
```